client/src/app.js

```diff
@@ -1477,7 +1467,9 @@ async function launch_genefusion(arg, app) {
 		const genome = app.genomes[arg.genome]
 		if (!genome) throw 'Invalid genome: ' + arg.genome
 
-		const m = await (await import('./fusion.parse')).parseFusion({
+		const m = await (
```

Comment:
@airenzp `await (await (...))` syntax is somewhat unusual, should follow the pattern elsewhere, for consistency:
```js
const _ = await import('./fusion.parse')
const m = await _.parseFusion({...})
```

Reply:
I will fix it, but just so you know, I did not edited these lines. There was some reformatting